pkg/report/linux.go

```diff
@@ -1634,8 +1635,22 @@ var linuxOopses = append([]*oops{
 				alt:    []string{"BUG: Dentry still in use [%[1]v of %[2]v]"},
 			},
 			{
-				title: compile("BUG: Bad page state"),
-				fmt:   "BUG: Bad page state",
+				title: compile("BUG: Bad page (state|cache)"),
+				fmt:   "BUG: Bad page %[1]v in %[2]v",
+				stack: &stackFmt{
+					parts: []*regexp.Regexp{
+						compile(`page last allocated`),
+						parseStackTrace,
+					},
+					parts2: []*regexp.Regexp{
```

Comment:
For x86_64 we use the frame where the page was allocated, but for arm64 we use the frame where the corruption was detected.
Does this mean we will have 2 duplicates for each bug?

Reply:
I've updated the code to only parse x86 stacks with page origins.